go/libraries/doltcore/schema/typeinfo/typeinfo.go

```diff
@@ -289,6 +289,8 @@ func FromKind(kind types.NomsKind) TypeInfo {
 		return Uint64Type
 	case types.UUIDKind:
 		return UuidType
+	case types.DecimalKind:
+		return DecimalDefaultType
```

Comment:
I'd return `&decimalType{sql.MustCreateDecimalType(65, 30)}` here as it's far wider than `10,5`

Reply:
Fixed https://github.com/dolthub/dolt/pull/1109/commits/290a79cc09c9da580ecd206528c9d8aae4095302